### PR TITLE
Set debug levels for OSM-debug handler

### PR DESCRIPTION
**Pull request by samueljohnsegun148**
This PR sets logging levels for OSM debug handler and will so handle issue  #711. Prior to now, OSM debug handler would simply check the incoming request. If the request was for a map, it would process, but would however return an invalid JSON response for a non-map case, which is not ideal. 

This is now fixed with the current debug levels implemented. So, in the event of a non-map request, it  won't still process but would return a valid JSON response together with the following debug information:
```23-06-09 22:09 UTC [DEBUG]: Received request
23-06-09 22:09 UTC [DEBUG]: Validating request
23-06-09 22:09 UTC [DEBUG]: Checking whether renderer is supported
23-06-09 22:09 UTC [DEBUG]: Checking for OpenStreetMap map data
23-06-09 22:09 UTC [INFO]: OSM map data not present. Skipping ...
23-06-09 22:09 UTC [DEBUG]: Sending response
```
And if the request is a map type, we would have something similar to this:
```
23-06-09 22:20 UTC [DEBUG]: Received request
23-06-09 22:20 UTC [DEBUG]: Validating request
23-06-09 22:20 UTC [DEBUG]: Checking whether renderer is supported
23-06-09 22:20 UTC [DEBUG]: Checking for OpenStreetMap map data
23-06-09 22:20 UTC [DEBUG]: Map data found! Processing data!
23-06-09 22:20 UTC [DEBUG]: Providing final result!
23-06-09 22:20 UTC [DEBUG]: Sending response
```

Please ensure you've followed the checklist and provide all the required information *before* requesting a review.
If you do not have everything applicable to your PR, it will not be reviewed!
If you don't know what something is or if it applies to you, ask!

Don't delete below this line.

---

## Required Information

- [x] I referenced the issue addressed in this PR.
- [x] I described the changes made and how these address the issue.
- [x] I described how I tested these changes.

## Coding/Commit Requirements

* [x] I followed applicable coding standards (e.g., [PEP8](https://pep8.org/))
* [x] I have not committed any models or other large files.

## New Component Checklist (**mandatory** for new microservices)

* [ ] I added an entry to `docker-compose.yml` and `build.yml`.
* [ ] I created A CI workflow under `.github/workflows`.

### CODE REVIEW

File: handlers/svg-open-street-map/osm-svg.py [hunk 1]
```diff
@@ -75,186 +123,210 @@ def handle():
...
-                                stroke_width=1.5,
-                                stroke='red'))
-                        all_svg.append(
-                            draw.Text(
-                                points_of_interest["cat"],
-                                16,
-                                longitude,
-                                latitude,
-                                fill='black'))
+            svg_layers = []
```
Comment (emmanuelwilson) [inline handlers/svg-open-street-map/osm-svg.py:153]: Some blocks of code like this one does not seem to align with the changes you've described in your PR. Let us know what else has been done in this PR so we can properly review it please! You only mention the change in debug messages (which are quite nice) but it seems like you've done more than just change the debug messages!

Comment (samueljohnsegun148) [reply handlers/svg-open-street-map/osm-svg.py:153]: Thanks, but nothing seems to be outside of what was described. The changes that look like some block of codes are not new additions, but appear so with the change of order/organization of the existing blocks brought by the introduction of the debug levels. For an example, in line 132 where the check for map or non-map data is taking place, the existing logic there would simply return an invalid JSON response (i.e. for non-map data), which is one major problem we are addressing with this PR, and it is addressed by the lines 134 -148. This however introduces the supposedly block change observed from line 149. A similar case for the debug level set at line 150 (that confirms if map data is found).

File: handlers/svg-open-street-map/osm-svg.py [hunk 2]
```diff
@@ -48,23 +58,61 @@ def handle():
...
         )
         validator.validate(contents)
     except ValidationError as e:
-        logging.error(e)
-        return jsonify("None"), 204
+        LOGGER.error(e)
+        return jsonify("Invalid request received!"), 400
 
     # Check preprocessor data
+    if "preprocessors" not in contents:
```
Comment (JRegimbal) [inline handlers/svg-open-street-map/osm-svg.py:71]: Why is this separate from the if statement directly below? This is a clear `else` state.

File: handlers/svg-open-street-map/osm-svg.py [hunk 3]
```diff
@@ -75,186 +123,210 @@ def handle():
...
             except jsonschema.exceptions.ValidationError as error:
-                logging.error(error)
+                LOGGER.error(error)
                 return jsonify("Invalid Preprocessor JSON format"), 500
-            logging.debug("Sending response")
+            LOGGER.debug("Sending response")
             return response
 
+        LOGGER.debug("Checking for OpenStreetMap map data ")
         if "ca.mcgill.a11y.image.preprocessor.openstreetmap"\
```
Comment (JRegimbal) [inline handlers/svg-open-street-map/osm-svg.py:132]: Same as above...this is just an else statement for the following if.

**Review by JRegimbal — CHANGES_REQUESTED** [review]
Confusing flow in some places, but otherwise seems good

File: handlers/svg-open-street-map/osm-svg.py (hunk 3 above)
Comment (samueljohnsegun148) [reply handlers/svg-open-street-map/osm-svg.py:132]: Well, I was only being careful not to add things that would be portrayed as new changes.  I've however made the changes now.

**Review by JRegimbal — CHANGES_REQUESTED** [review]

File: handlers/svg-open-street-map/osm-svg.py [hunk 4]
```diff
@@ -325,7 +325,7 @@ def handle():
                 LOGGER.debug("Failed to generate a valid response")
                 LOGGER.error(e)
                 return jsonify("Failed to generate a valid response"), 500
-            LOGGER.debug("Sending response")
+            LOGGER.debug("Finalresponse")
```
[the remark this answers is not in the document]
Comment (samueljohnsegun148) [reply handlers/svg-open-street-map/osm-svg.py]: Suggested change effected.

**Review by JRegimbal — APPROVED** [review]

**Pull request merged**